packages/mail-rendering/.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": "@comet/eslint-config/react",
+    "ignorePatterns": ["src/*.generated.ts", "lib/**"],
+    "rules": {
+        "@next/next/no-html-link-for-pages": "off", // disabled because lib has no pages dir
```

Comment:
```suggestion
```

This rule doesn't exist in the React config.

Reply:
a3a89e7d91d95e237e5bc14b361398d5bd2499ba